src/components/_home/_Home.tsx

```diff
@@ -49,7 +59,18 @@ export function _Home() {
             </Text>
           ) : (
             <>
-              <div className={classes.tableTitle}>Last testbooks</div>
+              <div className={classes.tableTitle}>
+                <div>Last testbooks</div>
+                <div>
+                  <button type="button" onClick={toggleTestsVisibility}>
```

Comment:
We can use the Mantine Button component here instead. There's a variant called transparent that would fit well.

We have something similar in the codebase already. See the delete button in the `CommentsList` component.

https://mantine.dev/core/button/